src/components/common/Back/styles.scss

```diff
@@ -0,0 +1,4 @@
+.button-back {
+  font-size: 2rem !important;
```

Comment:
It's not a good practice to use important. If you have to it means your style is overwriting something else.